code/game/atoms_movable.dm

```diff
@@ -686,3 +686,14 @@
 //Returns an atom's power cell, if it has one. Overload for individual items.
 /atom/movable/proc/get_cell()
 	return
+
+/atom/movable/proc/can_be_pulled(user)
+	if(src == user || !isturf(loc))
+		return FALSE
+	if(anchored || throwing)
+		return FALSE
+	if(isliving(src))
+		var/mob/living/L = src
+		if(L.buckled && L.buckled.buckle_prevents_pull)
+			return FALSE

```

Comment:
Do what @AnturK said: Remove these 4 lines. Create `/mob/living/can_be_pulled()` in `code/modules/mob/living/living.dm` and fill it out like this:

```
/mob/living/can_be_pulled()
    return ..() && !(buckled && buckled.buckle_prevents_pull)
```